pkg/rdkafka/NoOpKeySerializer.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Enqueue\RdKafka;
+
+class NoOpKeySerializer implements KeySerializer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function toString($key)
+    {
+        return $key;
```

Comment:
maybe we should force casting to a string here

Reply:
That would not be a `NoOp` anymore IMO, since casting would actually change they type. I would rather just keep the current behaviour and just pass the key through by default. What do you think?

Reply:
Than it contradicts to the interface contract. The KeySerializer toString methods returns string and I as a client should be able to rely on it. 

Also the rdkafka producer topic declares a key as a string 

https://arnaud-lb.github.io/php-rdkafka/phpdoc/class.rdkafka-producertopic.html

Reply:
OK, I agree. But then I'd rather rename it to `DefaultSerializer` to make it clear that there might be a type conversion.

Reply:
> OK, I agree. But then I'd rather rename it to DefaultSerializer to make it clear that there might be a type conversion.

Sounds good.